src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/Models/BatchReadFileHeaders.cs

```diff
@@ -11,26 +11,29 @@
 namespace Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models
 {
     using Newtonsoft.Json;
-    using System.Collections;
-    using System.Collections.Generic;
     using System.Linq;
 
-    public partial class RecognitionResult
+    /// <summary>
+    /// Defines headers for BatchReadFile operation.
+    /// </summary>
+    public partial class BatchReadFileHeaders
     {
         /// <summary>
-        /// Initializes a new instance of the RecognitionResult class.
+        /// Initializes a new instance of the BatchReadFileHeaders class.
         /// </summary>
-        public RecognitionResult()
+        public BatchReadFileHeaders()
```

Comment:
This is a breaking change, please bump the Nuget version to `4.0.0` and update the csproj/AssemblyInfo.cs files

Reply:
Thanks. Nuget version bumped to `4.0.0`.

Besides, I think GIT diff is showing a wrong comparison here. 

`Models/RecognitionResult.cs` → `Models/TextRecognitionResult.cs`
`Models/BatchReadFileHeaders.cs` is a newly introduced file.